### PR TITLE
added removefromqueue

### PR DESCRIPTION
**Pull request by Fuechschen**
This adds #2. Method is tested and works.

### CODE REVIEW

**Comment by anjanms** [comment]
Great. I tested it as well, it works. :+1: 

Could you please take care of these two things?
- Remove the form -- It's not needed for this request.
- Change the method name to ```moderateRemoveDJ``` -- The current one is a bit wordy, it also makes things easier for people porting bots over.

Thanks!

**Comment by Fuechschen** [comment]
#done
